spezi-module/contact/src/test/java/edu/stanford/spezikt/spezi_module/contact/ContactRepositoryTest.kt

```diff
@@ -0,0 +1,34 @@
+package edu.stanford.spezikt.spezi_module.contact
+
+import edu.stanford.spezikt.spezi_module.contact.model.ContactOptionType
+import edu.stanford.spezikt.spezi_module.contact.repository.DefaultContactRepository
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+
+class DefaultContactRepositoryTest {
+
+    private lateinit var repository: DefaultContactRepository
+
+    @Before
+    fun setup() {
+        repository = DefaultContactRepository()
+    }
+
+    @Test
+    fun defaultContactRepository_loadsContact() {
+        val contact = repository.getContact()
+        assertEquals(contact.name, "Leland Stanford")
```

Comment:
I would propose using [Truth](https://truth.dev/) as assertion library. It has a pretty nice api, `assertThat(contact.name).isEqualTo("Leland Stanford")`

Reply:
Seems to be more readable. I like this. I changed to that lib